app/assets/stylesheets/modules/_family.scss

```diff
@@ -131,3 +127,66 @@
   }
 }
 
+.family-names {
+  text-align: center;
```

Comment:
Properties should be ordered background-color, border-radius, margin, padding, text-align